R/rtmidi/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "rtmidi"
+version = v"5.0.0"
```

Comment:
I think that it should be `v"4.0.0"` .  https://github.com/thestk/rtmidi/blob/6e4e763/RtMidi.h#L61